src/hooks/characterSheet/useStatAndModifier.test.tsx

```diff
@@ -1,13 +1,23 @@
-import { describe, expect, it } from "vitest";
+import { afterEach, describe, expect, it } from "vitest";
 import { renderHook } from "@testing-library/react";
 import { useStatAndModifier } from "./useStatAndModifier";
 import { createStoreWrapper } from "@/test-utils";
 
 describe("useStatAndModifier", () => {
+	let cleanup: (() => Promise<void>) | undefined;
+
+	afterEach(async () => {
+		if (cleanup) {
+			await cleanup();
+			cleanup = undefined;
+		}
+	});
+
 	it("retrieves the correct stat and modifier", async () => {
 		const characterSheetId = "test-character-sheet";
 
-		const { wrapper } = await createStoreWrapper();
+		const { wrapper, cleanup: testCleanup } = await createStoreWrapper();
+		cleanup = testCleanup;
```

Comment:
Don't use a global cleaner but use onTestFinished from the context, which I want as a required parameter to createStoreWrapper. This way, that function can clean after itself. 

Example: 
```
import { test } from 'vitest'

test.concurrent('performs a query', ({ onTestFinished }) => {
  const db = connectDb()
  onTestFinished(() => db.close())
  db.query('SELECT * FROM users')
})
```